example/docker-compose/distributed/readme.md

```diff
@@ -0,0 +1,66 @@
+## Local Storage
+
+In this example all data is stored locally in the `tempo-data*` directories for each of the ingester instances. Local storage is fine for experimenting with Tempo
+or when using the single binary, and each instance mounts its own data directory to keep the cluster resources separated.
```

Comment:
I believe this is still copied from the local example. You can probably copy some text from the s3 example and mention that with this setup all components run independently, which is more scalable and fault-tolerant.

Reply:
I'd also mention this is the distributed setup, otherwise it's almost the same as the s3 example.

> _In this example tempo is configured to write data to S3 via MinIO which presents an S3 compatible API._

⬇️ 

> _In this example Tempo is deployed in distributed mode and writes data to S3 via MinIO, in distributed mode the Tempo components are deployed as microservices. The microservices deploy of Tempo is fault tolerant, high volume, independently scalable._

What do you think?

Reply:
That sounds good to me.  I've made the update to include some verbiage about being distributed.